torchx/schedulers/test/aws_batch_scheduler_test.py

```diff
@@ -127,7 +127,7 @@ def test_submit_dryrun_with_priority(self) -> None:
     def test_submit_dryrun(self) -> None:
         scheduler = create_scheduler("test")
         app = _test_app()
-        cfg = AWSBatchOpts({"queue": "testqueue"})
+        cfg = AWSBatchOpts({"queue": "testqueue", "user": "testuser"})
```

Comment:
might be nice to have a separate test just for tags but not a big deal

Reply:
done